### ISSUE TITLE
Tell the algorithm when it found something that's not a face

### ISSUE THREAD

**Issue by scepterus** [LibrePhotos/librephotos#848]
**Describe the enhancement you'd like**
Right now when I scan for faces I get objects that are mistaken for faces a lot. if there was a way to flag them as not a face that would help a lot.

**Describe why this will benefit the LibrePhotos**
It would make the training faster, and better if you can tell it, this is not a face, and this is not this person. And then the next time it trains, it would know not to make those mistakes again.

**Additional context**
Add any other context or screenshots about the enhancement request here.

**Comment by gitdoc67** [comment]
Agreed. Also the ability to ignore a photo from facial recognition. I have some group photos with dozens or hundreds of people and dont want to bother tagging all of them.

**Comment by derneuere** [comment]
Implemented — the "delete face" action sets `Face.deleted`, and every face queryset filters on it, so a flagged non-face will not resurface in the unknown/inferred/clustering lists.
